test/apps/iodemo/io_demo.cc

```diff
@@ -811,7 +811,8 @@ class P2pDemoCommon : public UcxContext {
                                          test_opts.chunk_size),
                            "data iovs"),
         _data_chunks_pool(test_opts.chunk_size, "data chunks",
-                          test_opts.memory_type, test_opts.num_offcache_buffers)
+                          test_opts.memory_type, test_opts.num_offcache_buffers),
+        _iov_buf_filler(iov_buf_filler)
```

Comment:
why need this in base class?
seems to me we need to define static const uint32_t members in both DemoCLient/DemoServer

Reply:
better to keep in base class, since it is common thing for both client and server classes

Reply:
looks weird to me because the base class does not use it..
might as well just pass 0xeeee / 0xddd directly to iov->init()

Reply:
> might as well just pass 0xeeee / 0xddd directly to iov->init()

but then it will be duplicated.
maybe is it worth just moving `_iov_buf_filler` to DemoServer/DemoClient?